beater/config/java_attacher.go

```diff
@@ -23,8 +23,9 @@ import "fmt"
 // attacher jarfile.
 type JavaAttacherConfig struct {
 	Enabled        bool                `config:"enabled"`
-	DiscoveryRules []map[string]string `config:"discovery_rules"`
+	DiscoveryRules []map[string]string `config:"discovery-rules"`
```

Comment:
After giving it another thought, I think we should remove the include `include-pid` and `exclude-pid` rules. I don't think they make sense in the context of globally specifying include/exclude rules. That's because PIDs are not predictable before a process starts and even more so across different hosts.

Reply:
makes sense :ok_hand: